src/core/src/op/divide.cpp

```diff
@@ -246,7 +246,7 @@ bool Divide::evaluate(TensorVector& outputs, const TensorVector& inputs) const {
                                       this,
                                       outputs,
                                       inputs,
-                                      OV_PP_ET_LIST(f32, i32, i64, u32, u64),
+                                      OV_PP_ET_LIST(f32, i8, i32, i64, u8, u32, u64),
```

Comment:
If these precisions are added to get reference values for tests the template plugin should be only updated.

Reply:
1) The template plugin uses evaluate methods of nodes. Why I can not update evaluate method if I need execution in `int8`? If still not, then where I should update template plugin?
2) `Multiply`, `Subtract` & `Add` operations supports execution in `int8`, why `Divide` doesn't support?

Reply:
> Why I can not update evaluate method if I need execution in int8
We are trying to minimize number of evaluate instances in Core to minimize binay size. In general we shouldn't introduce implementations in Core which are required for test only.
> If still not, then where I should update template plugin?
See  https://github.com/openvinotoolkit/openvino/pull/22649 for refernce (template plugin changes).

However I would agree with that point that other math operations like Add,Mul etc has i8/u8 evaluates. So we need to align here first. @praasz

Reply:
The main concern is binary size.
The evaluate in core operations are required for constant folding, shape inference and as a fallback path for plugins. If certain precision is not required to be implemented because of mentioned reasons then it should not be added in core operator.

For testing purposes the TEMPLATE plugin can be expanded to support any required precision as it is used to calculate reference values. It uses evaluate from operator and if precision is not implemented or there is no evaluate at all then it uses plugin implementation.

The TEMPLATE evaluates are implemented in `src/plugins/template/backend/ops/`
Use this `bitwise_or` as reference [src/plugins/template/backend/ops/bitwise_or.cpp](https://github.com/praasz/openvino/blob/be0c954fd212005006abffc15c50d267d9a04b94/src/plugins/template/backend/ops/bitwise_or.cpp).

If there is no files for required operators just add them with required precisions. Check also this file to register TEMPLATE implementation `src/plugins/template/backend/opset_int_tbl.hpp`

If this PR is urgent to merge so I can accept but just make follow up task to clean it as next step.